src/NuGet.Core/NuGet.CommandLine.XPlat/Program.cs

```diff
@@ -62,25 +63,27 @@ public static int Main(string[] args)
                     "-s|--source <source>",
                     Strings.Restore_Switch_Source_Description,
                     CommandOptionType.MultipleValue);
+
                 var packagesDirectory = restore.Option(
                     "--packages <packagesDirectory>",
                     Strings.Restore_Switch_Packages_Description,
                     CommandOptionType.SingleValue);
-                var parallel = restore.Option(
-                    "-p|--parallel <noneOrNumberOfParallelTasks>",
-                    string.Format(
-                        CultureInfo.CurrentCulture,
-                        Strings.Restore_Switch_Parallel_Description,
-                        RestoreRequest.DefaultDegreeOfConcurrency),
-                    CommandOptionType.SingleValue);
+
+                var disableParallel = restore.Option(
```

Comment:
This kind of decreases the amount of control the user has. Is this consistent with the other client experiences?